### ISSUE TITLE
[CSS][A11y] Needs focus styling and labeling

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/reactsocialnetwork#2]
**Describe the bug**
[Accessibility]
 Focus Styling is missing for icons in navbar.
 No labels for icons.

**To Reproduce**

Run the dev server with npm start in the root dir. After the app loads in browser,
Press Ctrl Shift J or otherwise load the browser console.

1. Go to 'Accessiblity'
2. Click on 'All issues' button next to "Check for issues:"
3. See errors which include Keyboard, Contrast and Text-leaf for the icons in the navbar and the 

**Expected behavior**

No focus styling issues with icons in navbar.  Consider using focus visible and aria labeling.

**Desktop (please complete the following information):**
   Linux, 64bit
   Firefox
   107

**Additional context**
This is an accessibility issue.
